hoomd/mpcd/CMakeLists.txt

```
@@ -58,6 +58,7 @@ set(_forces
 
 set(_geometries
     ParallelPlateGeometry
+    ConcentricCylindersGeometry
```

Please add files alphabetically! It makes it easier to ensure they are there.
```suggestion
    ConcentricCylindersGeometry
    ParallelPlateGeometry
```